src/schema/v2/partner/Settings/repositionPartnerLocations.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  GraphQLString,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLUnionType,
+  GraphQLList,
+} from "graphql"
+import { mutationWithClientMutationId } from "graphql-relay"
+import {
+  formatGravityError,
+  GravityMutationErrorType,
+} from "lib/gravityErrorHandler"
+import { ResolverContext } from "types/graphql"
+import { PartnerType } from "../partner"
+
+interface RepositionPartnerLocationsProps {
+  partnerId: string
+  locationIds: string[]
+}
+
+const SuccessType = new GraphQLObjectType<any, ResolverContext>({
+  name: "RepositionPartnerLocationsSuccess",
+  isTypeOf: (data) => !!data._id,
+  fields: () => ({
+    partner: {
+      type: PartnerType,
```

Comment:
Think its cool to return the `PartnerType` here?

Reply:
It depends on how flexible you want it! Sometimes its very very useful to do it like this, if for example you wanna refresh some parent component tree by spreading in `...SomeRoute_partner` in yr mutation at the consumer callsite. 

Personally i'd defer to wider return types like this for flexibility, but in certain performance-sensitive contexts you'd might just want to hit a subsection of the data.

Reply:
https://github.com/artsy/metaphysics/pull/6609#discussion_r2034127949

Whatever you need/want. You can even put 'the universe' here (😆 ) -  `partner` makes sense to me

Reply:
much power and much responsibility

thanks for talking it out 😄 will go with `partner`!